packages/common/src/components/CondaPkgList.tsx

```diff
@@ -248,6 +198,24 @@ class CondaPkgView extends React.Component<IPkgListProps> {
   protected rowRenderer = (props: ListChildComponentProps): JSX.Element => {
     const { data, index, style } = props;
     const pkg = data[index] as Conda.IPackage;
+    const isDataRow = index < data.length;
+
+    // When the row index is one greater than the last index of the data array,
+    // it's time to render a "Loading..." indicator row
+    if (!isDataRow) {
```

Comment:
There's a little sliver of doubt in my mind about what happens when you really get to the end of the data. Does this function still get called and attempt to render a "Loading..." row even when there's nothing left to load?

Reply:
You know, I'm not sure. If the `FixedSizeList` has the data as part of its props, why would `index` ever exceed `data.length`? I've seen this pattern before, but I have no idea why this check is necessary. It seems to me like it should be the job of `FixedSizeList` to not try to render data it doesn't have :shrug:

Reply:
Well, I think I follow the logic inside `render()` below, and it seems to me like `index` should only be able to be greater than `index.length` in the case where there's more pages of data to fetch. This seems fine to me - but I guess we can just test it to see if it works?